src/azul/portal_service.py

```diff
@@ -41,6 +43,20 @@ def __init__(self):
         self.client = aws.client('s3')
         self.version_service = VersionService()
 
+    @property
+    def bucket(self):
+        return config.portal_db_bucket
+
+    @property
+    def object_key(self):
+        return config.portal_db_object_key(self.data_source)
+
+    @cached_property
+    def data_source(self):
+        # FIXME https://github.com/DataBiosphere/azul/issues/2716
+        catalog = config.default_catalog
```

Comment:
I think this variable could be inlined, up to you. If the way it is now is clearer, disregard my comment.

Reply:
That's the idea, although mostly I was following Hannes's precedent from https://github.com/DataBiosphere/azul/blob/dc202ee2c273cee1bff823ee2b930d5d8fe0addc/src/azul/portal_service.py#L155